RELEASE_NOTES.md

```diff
@@ -1,104 +0,0 @@
-# Release Notes
```

Comment:
Why is this being deleted?

Reply:
It seems redundant. I can bring it back, but I migrated all the notes to [Github](https://github.com/quintilesims/layer0/releases)

Reply:
Ah cool, didn't realize that 👍